src/Appwrite/Utopia/Response/Model/Func.php

```diff
@@ -145,6 +145,18 @@ public function __construct()
                 'default' => false,
                 'example' => false,
             ])
+            ->addRule('memory', [
+                'type' => self::TYPE_INTEGER,
+                'description' => 'Function execution memory limit in MB.',
+                'default' => 512,
+                'example' => 1024,
+            ])
+            ->addRule('cpus', [
+                'type' => self::TYPE_INTEGER,
```

Comment:
Let's confirm with team lead.. Should this be integer, or string? If it's integer I would expect slider on Console. If it was string, I would expect select dropdown.
Same for developers using API, integer indicates I might be able to set it to anything. Using ENUM here might work better

Reply:
Enum will probably make more sense here, will update appropiately

Reply:
Can we add enums to models? I can't seem to see a `self::TYPE_ENUM` in the Model class

Reply:
Not sure, if we never did it, we can do STRING. Check what we did for `plan` in billing, I think that behaves as enum